test/celeritas/field/LinearPropagator.test.cc

```diff
@@ -25,35 +25,19 @@ namespace test
 // TEST HARNESS
 //---------------------------------------------------------------------------//
 
-template<class HP>
-class LinearPropagatorTest : public AllGeoTypedTestBase<HP>
+class LinearPropagatorTest : public CoreGeoTestBase
```

Comment:
This change means that we are only testing one geometry type per build, rather than all three. I think that can be relaxed later but is fine for the time being.